scripts/src/models/translations.ts

```diff
@@ -30,13 +30,13 @@ export class OfferTranslation extends BaseEntity {
 		}
 
 		const query = OfferTranslation.createQueryBuilder("translations");
-		if (languages) {
+		if (languages && languages.length) {
```

Comment:
Please be specific and do `languages.length > 0`

Reply:
Done

Reply:
What!? no! This very idiomatic to JS and concise.